itest

```diff
@@ -4,10 +4,10 @@ set -euo pipefail
 path="$1"
 
 apt-get update
-apt-get -y install --no-install-recommends gdebi-core software-properties-common
+apt-get -y install --no-install-recommends software-properties-common
 add-apt-repository ppa:deadsnakes/ppa
 
 apt-get update
-gdebi -n "$path"
+apt install -y "$path"
```

Comment:
`apt` is not intended for scripting purposes and doesn't actually test the package installation

Reply:
and actually, if `gdebi` is crashing then that means the `.deb` is malformed -- switching to `apt` is hiding the bug

Reply:
Yeah just noticed the big huge
`WARNING: apt does not have a stable CLI interface yet. Use with caution in scripts.`
When looking at the travis logs. Definitely don't want that.

Switched back to `gdebi` and added the build with `gzip` override.